packages/manager/apps/dedicated/client/app/account/identity-documents/translations/Messages_fr_FR.json

```
@@ -17,7 +17,7 @@
   "user_account_identity_documents_selection_file_format_invalid": "Format de fichier non pris en charge. Veuillez utiliser l'un des formats acceptés : jpg, jpeg, pdf et png",
   "user_account_identity_documents_submit": "Envoyer mes documents",
   "user_account_identity_documents_submit_success_message_title": "Merci, nous avons bien reçu vos documents!",
```

using exclamation point without a previous space is fairly informal

```suggestion
  "user_account_identity_documents_submit_success_message_title": "Merci, nous avons bien reçu vos documents !",
```